### PR TITLE
(Feature) Support of Sokol network

### PR DESCRIPTION
**Pull request by vbaranov**
Closes https://github.com/poanetwork/metamask-extension/issues/3
Relates to #5
Relates to #8

- RPC endpoint
- Sokol faucet
- link to POA Sokol explorer for txs and address
- 0 exchange rate for Sokol
- SPOA name of the coin for Sokol

Contains changes from https://github.com/poanetwork/metamask-extension/pull/7

screen shot 2018-07-19 at 16 43 26
screen shot 2018-07-19 at 16 43 15
screen shot 2018-07-19 at 16 43 01
screen shot 2018-07-19 at 16 42 39
screen shot 2018-07-19 at 16 42 28
screen shot 2018-07-19 at 16 42 20

### CODE REVIEW

**Comment by vbaranov** [comment]
Screenshots were updated after implementing zero exchange rate for Sokol and name of the coin (SPOA)

File: app/scripts/controllers/network/network.js [hunk 1]
```diff
@@ -15,8 +15,10 @@ const {
   KOVAN,
   MAINNET,
   LOCALHOST,
+  POA_SOKOL,
```
Comment (fvictorio) [inline app/scripts/controllers/network/network.js]: I can't comment in that line so I'll do it here:

Maybe the default provider should be Sokol when in testMode. See line 29 below:

```
const defaultProviderConfig = {
  type: testMode ? RINKEBY : MAINNET,
}
```

Comment (vbaranov) [reply app/scripts/controllers/network/network.js]: Yes, agree! I'll fix it. Thanks.

Comment (vbaranov) [reply app/scripts/controllers/network/network.js]: @fvictorio fixed

**Comment by vbaranov** [comment]
>I'd say we merge this first. That will generate merge conflicts with my PR. When that happens, I will fix them and add the things that is missing.
@fvictorio are you ok with this option? if yes, let's do it this way.

@patitonar do you have any concerns about changes in this PR?

**Comment by fvictorio** [comment]
> @fvictorio are you ok with this option? if yes, let's do it this way.

Sure, I created #12.

**Pull request merged**